src/orquestra/sdk/_base/_environment.py

```diff
@@ -0,0 +1,5 @@
+CONFIG_PATH_ENV = "ORQ_CONFIG_PATH"
+DB_PATH_ENV = "ORQ_DB_PATH"
+RAY_TEMP_PATH_ENV = "ORQ_RAY_TEMP_PATH"
+RAY_STORAGE_PATH_ENV = "ORQ_RAY_STORAGE_PATH"
+RAY_DOWNLOAD_GIT_IMPORTS_ENV = "ORQ_RAY_DOWNLOAD_GIT_IMPORTS"
```

Comment:
**suggestion**: this module is super cool! Could you also add here the passport env variable?

Reply:
Done 4d5fa6c